### PR TITLE
fix audit backends decription

### PR DESCRIPTION
**Pull request by charrywanganthony**

### CODE REVIEW

**Comment by bradtopol** [comment]
Hi @liggitt can you do a technical review for this PR? Simply add a /lgtm if it is technically accurate.

File: content/en/docs/tasks/debug-application-cluster/audit.md [hunk 1]
```diff
@@ -92,15 +92,16 @@ admins constructing their own audit profiles.
...
 Audit backends persist audit events to an external storage.
-[Kube-apiserver][kube-apiserver] out of the box provides two backends:
+[Kube-apiserver][kube-apiserver] out of the box provides three backends:
 
 - Log backend, which writes events to a disk
 - Webhook backend, which sends events to an external API
 - Dynamic backend, which configures webhook backends through an AuditSink API object.
 
-In both cases, audit events structure is defined by the API in the
+In top two cases, audit events structure is defined by the API in the
```
Comment (liggitt) [inline content/en/docs/tasks/debug-application-cluster/audit.md]: change to "In all cases, ..."

File: content/en/docs/tasks/debug-application-cluster/audit.md [hunk 2]
```diff
@@ -92,15 +92,16 @@ admins constructing their own audit profiles.
...
 `audit.k8s.io` API group. The current version of the API is
-[`v1`][auditing-api].
+[`v1`][auditing-api]. But in dynamic backend, AuditSink objects are defined in
+`auditregistration.k8s.io` API group. The current version of the API is v1alpha1.
```
Comment (liggitt) [inline content/en/docs/tasks/debug-application-cluster/audit.md]: the auditsink registration is separate from the format of the audit events sent to the sink. in all cases, the events sent are defined by the API in the `audit.k8s.io` API group.

Comment (charrywanganthony) [reply content/en/docs/tasks/debug-application-cluster/audit.md]: Thanks @liggitt , modified.

**Comment by mistyhacks** [comment]
/approve

@charrywanganthony In future, can you please provide a PR description, even if it is something like "Update description of defaults to match reality". It helps reviewers.

**Pull request merged**